torchbenchmark/models/alexnet/__init__.py

```diff
@@ -3,16 +3,27 @@
 import torch
 import torch.optim as optim
 import torchvision.models as models
+from ...util.model import BenchmarkModel
 
-class Model:
+class Model(BenchmarkModel):
     def __init__(self, device="cpu", jit=False):
+        super().__init__()
         self.device = device
         self.jit = jit
         self.model = models.alexnet()
         if self.jit:
             self.model = torch.jit.script(self.model)
         self.example_inputs = (torch.randn((32, 3, 224, 224)),)
 
+    def eval(self):
+        self.set_mode(self, False)
+    
+    def train(self):
+        self.set_mode(self, True)
+
+    def set_mode(self, train = True):
+        self.model.train(train)
```

Comment:
yes, i think this is great.  I guess it will work out of the box for most models, without changing the model.  The key part is to carefully check if any model does not work this way, and make sure to fix it;  

Also, can you update the 'Adding_models.md' so that anyone following the instructions knows to make sure they implement this if they need to.